### PR TITLE
feat - added alerts and alert docs

### PR DESCRIPTION
**Pull request by Ayesha-7**

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
Good to merge

File: Styles/Components/alert.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+.alert-icon {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+}
+
+.alert-icon .icon {
+  width: 1.5rem;
+  height: 1.5rem;
+  margin-right: 0.5rem;
```
Comment (Prabhav-Sharma) [inline Styles/Components/alert.css]: Use global variables for spacing properties like margin, padding, gap to maintain consistency across the web app

Comment (Kavita539) [inline Styles/Components/alert.css]: sugesstion: Global variables can be used for spacings

**Review by Kavita539 — COMMENTED** [review]
Overall looks good

**Review by Prabhav-Sharma — APPROVED** [review]

**Review by Kavita539 — APPROVED** [review]

File: Docs/alert.html [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+          <h3 class="mb-1">Alert with Icon</h3>
+          <div>
+            <p>
+              Use classes alert & alert-icon to create alert with icons. Refer
+              the example given below.
+            </p>
+          </div>
+          <div class="component-container my-2 p-2">
+            <div class="alert alert-info alert-icon my-05">
+              <img class="icon" src="../assets/images/info-button.png" />
```
Comment (deepak29-git) [inline Docs/alert.html:177]: add alt attribute so that if image is loaded then show the alternative text

File: Styles/variables.css [hunk 3]
```diff
@@ -16,5 +23,41 @@
...
   --blue-1: #0000ff;
   --blue-2: #0080ff;
-  --blue-5: #0084ff34;
+  --blue-3: #69c0ff;
+  --blue-4: #91d5ff;
+  --blue-5: #bae7ff;
+  --blue-6: #e6f7ff;
+  /* --blue-5: #0084ff34; */
+
+  /* will clean it later if not used */
```
Comment (deepak29-git) [inline Styles/variables.css:32]: remove unnecessary code

**Review by deepak29-git — COMMENTED** [review]
rest of looks good to me

**Pull request merged**